src/zopfli/katajainen.c

```diff
@@ -172,9 +172,27 @@ static void ExtractBitLengths(Node* chain, Node* leaves, unsigned* bitlengths) {
 
 /*
 Comparator for sorting the leaves. Has the function signature for qsort.
+Mr_KrzYch00's Fix:
```

Comment:
A 1-line comment "keep sort stable" or similar would be sufficient.
Your name can go to the CONTRIBUTERS file. Please also read CONTRIBUTING.md.